python/data/ArtusConfigs/Includes/settingsKappa.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+import Artus.Utility.logger as logger
+log = logging.getLogger(__name__)
+
+import re
+import json
+import copy
+import Artus.Utility.jsonTools as jsonTools
+import Kappa.Skimming.datasetsHelperTwopz as datasetsHelperTwopz
+import importlib
+
+def build_config(nickname):
+  config = jsonTools.JsonDict()
+  datasetsHelper = datasetsHelperTwopz.datasetsHelperTwopz("Kappa/Skimming/data/datasets.json")
+  
+  
+  ## fill config:
+  # includes
+  includes = [
+    ]
+  for include_file in includes:
+    analysis_config_module = importlib.import_module(include_file)
+    config += analysis_config_module.build_config(nickname)
+  
+  # explicit configuration
+  config["GenParticles"] = "" if re.search("(?<!PFembedded).Run201", nickname) else "genParticles"
```

Comment:
This statement is used quite often. How about a "isMC" flag instead of the regex?

Reply:
The flag is now included in the recent commit